archive/test/ArchiveControllerTest.scala

```diff
@@ -1,21 +1,30 @@
 package test
 
+import akka.actor.ActorSystem
+import common.ExecutionContexts
 import controllers.ArchiveController
+import model.{ApplicationContext, ApplicationIdentity}
 import play.api.mvc.Result
 import play.api.test.Helpers._
 import org.scalatest.{BeforeAndAfterAll, DoNotDiscover, FlatSpec, Matchers}
+import play.api.Environment
+import rendering.core.Renderer
 
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
 import services.RedirectService
 import services.RedirectService.{ArchiveRedirect, PermanentRedirect}
 
 @DoNotDiscover class ArchiveControllerTest
   extends FlatSpec
   with Matchers
   with ConfiguredTestSuite
+  with ExecutionContexts
   with BeforeAndAfterAll {
 
-  lazy val archiveController = new ArchiveController(mockRedirects)
+  implicit lazy val appContext: ApplicationContext = ApplicationContext(Environment.simple(), ApplicationIdentity("archive"))
+  implicit lazy val actorSystem: ActorSystem = app.actorSystem
+  lazy val renderer = new Renderer
```

Comment:
I will probably abstract the creation of the test renderer since we will need to add it for all controller tests.